packages/bootstrap-4/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -24,9 +25,7 @@ const FieldTemplate = ({
       required={required}>
       {children}
       {displayLabel && rawDescription ? (
-        <Typography variant="caption" color="textSecondary">
-          {rawDescription}
-        </Typography>
+        <Form.Text>{rawDescription}</Form.Text>
```

Comment:
Does this element need to be styled as in the material-ui example?

Fine putting a todo here if needed and rolling back later